lib/broccoli_sass_compiler.js

```diff
@@ -493,14 +489,18 @@ module.exports = class BroccoliSassCompiler extends BroccoliPlugin {
       // populate the output cache for rebuilds
       this.addOutput(details.fullSassFilename, outputFile);
 
-      mkdirp.sync(path.dirname(cachedFile));
-      fs.writeFileSync(cachedFile, new Buffer(data, "base64"));
-
-      mkdirp.sync(path.dirname(outputFile));
-      ensureSymlink(cachedFile, outputFile);
+      this.writeDataToFile(cachedFile, outputFile, new Buffer(data, "base64"));
     });
   }
 
+  writeDataToFile(cachedFile, outputFile, data) {
```

Comment:
I'n general, functions that don't have need `this` (and are not hooks for subclasses) should likely be pure functions at the top of the module, rather then being on the prototype.

I would recommend fleshing this idea out further.

Reply:
I updated this to move that function to the top of the module